domain/src/main/java/tachiyomi/domain/library/model/LibrarySortMode.kt

```diff
@@ -31,6 +31,7 @@ data class LibrarySort(
         data object ChapterFetchDate : Type(0b00011000)
         data object DateAdded : Type(0b00011100)
         data object TrackerMean : Type(0b000100000)
+        data object Random : Type(0b000100100)
```

Comment:
I'm not well versed in bit masking but the mask probably needs to be changed?

Reply:
Up to you! All that's going on here is that the bits in the mask that are one, e.g.

`override val mask: Long = 0b00111100L`

are relevant, everything else is ignored. What this means practically is that we only care about the four numbers in the middle and not the outer two on either side. These seem to just ascending integers in binary - 0, 1, 10, 11, 100, etc. The last one we have currently is TrackerMean at 1000 (8, as it's the 8th indexed element), so I put Random at 9 (1001). Factoring in the mask that's 0b00100100 - though I'm just now noticing that some of these have an extra leading zero, so I'll add a fix for that. Random is definitely arguably set apart from the rest, though, so if we wanted to make it 0b00111100 that wouldn't be a bad idea either.

Reply:
There are extra sort items in some of the forks and SY for example went with 00100100, same as here

Reply:
Oh, also, unless I'm very wrong, removing the leading zero on the previous sort shouldn't change the value since it's leading, it'll just step it down to a eight bit value (which it is almost certainly being squashed to anyway when the flag is composed)

Reply:
> Random is definitely arguably set apart from the rest, though, so if we wanted to make it 0b00111100 that wouldn't be a bad idea either.

Fair point. Bump it to max